prime-angular-frontend/src/app/modules/enrolment/pages/care-setting/care-setting.component.ts

```diff
@@ -14,6 +14,7 @@ import { AuthService } from '@auth/shared/services/auth.service';
 import { PermissionService } from '@auth/shared/services/permission.service';
 import { Role } from '@auth/shared/enum/role.enum';
 
+import { HealthAuthorityEnum } from '@lib/enums/health-authority.enum';
```

Comment:
This one too please

Reply:
Ops! Also, removed other unused imports.